r2/r2/templates/linkinfobar.html

```diff
@@ -39,7 +39,7 @@
       &#32;
     </span>
     <time datetime="${html_datetime(thing.a._date)}">
-      ${thing.a._date.strftime(thing.datefmt)}
+      ${thing.a._date.strftime(_('%d %b %Y'))}
```

Comment:
`c.locale` should contain the current locale. See [other users of the babel functions for examples](https://github.com/reddit/reddit/blob/master/r2/r2/lib/template_helpers.py#L586-L590).